harness/src/dev_harness/validators.py

```
@@ -0,0 +1,92 @@
+        text = path.read_text(encoding="utf-8", errors="ignore")
+        match = re.search(r"^feature_id:\s*(\d{3,})\s*$", text, re.MULTILINE)
+        if not match:
+            errors.append(f"{path}: missing numeric feature_id")
+            continue
+        feature = match.group(1)
+        if feature in seen:
+            errors.append(f"{path}: duplicate feature_id {feature}")
+        seen.add(feature)
+        if path.stem != f"F{feature}":
+            errors.append(f"{path}: filename must match feature_id")
+        if not re.search(r"^summary:\s*.+[А-Яа-яЁё]", text, re.MULTILINE):
```

Ruff (0.16.2)

[warning] String contains ambiguous `А` (CYRILLIC CAPITAL LETTER A). Did you mean `A` (LATIN CAPITAL LETTER A)?

(RUF001)